deploy/helm/stacks-dotnet/templates/secret.yaml

```diff
@@ -9,4 +9,10 @@ metadata:
   namespace: {{ .Values.namespace }}
 type: Opaque
 data:
+  {{- if eq .Values.cloudProvider "azure" }}
   app_insights: {{ .Values.pod.appInsightsKey | b64enc | quote }}
+  {{- end }}
+  {{- if eq .Values.cloudProvider "aws" }}
+  CLOUDWATCH_LOG_GROUP: {{ .Values.pod.cloudwatchLogGroup | b64enc | quote  }}
+  CLOUDWATCH_STREAM_PREFIX: {{.Values.pod.cloudwatchStreamPrefix | b64enc | quote }}
```

Comment:
Is the app currently logging to CW directly? We have Container Insights installed on the cluster which ships all stdout to CW automatically, unless these are extra metrics or something

Reply:
Doesn't look like it is used on the app from searching for those environment variables. I was just converting what was already in the Kubernetes manifests so didn't check. Would you expect to find the logs in this group /aws/eks/ensono-stacks-nonprod-ew2-eks/cluster? I can't see any for the dotnet api

Reply:
They're under `/aws/eks/ensono-stacks-nonprod-ew2-eks/application`
![image](https://github.com/user-attachments/assets/747cce12-9166-4165-b373-b7cd80aae490)

> I was just converting what was already in the Kubernetes manifests so didn't check

Yeah no worries, those manifests are all very old, as they were before we moved the cluster to the new one, which has Enhanced CI installed and nginx (hence the AWS LB Controller annotations in the Ingress that you removed)

I'm wondering if they're even needed any more, as we ship all stdout?

Reply:
Fair enough, I'll remove them a check we're still getting logs. We can always add them back if needed

Reply:
Cool cool

Reply:
Nice nice, thanks for confirming 👍🏻